gensim/test/test_nmf.py

```diff
@@ -139,15 +160,15 @@ def testLargeMmapCompressed(self):
         self.assertRaises(IOError, nmf.Nmf.load, fname, mmap='r')
 
     def testDtypeBackwardCompatibility(self):
-        nmf_3_6_0_fname = datapath('nmf_3_6_0_model')
+        nmf_fname = datapath('nmf_model')
         test_doc = [(0, 1), (1, 1), (2, 1)]
-        expected_topics = [(0, 1.0)]
+        expected_topics = [(1, 1.0)]
 
         # save model to use in test
-        self.model.save(nmf_3_6_0_fname)
+        # self.model.save(nmf_fname)
```

Comment:
Why is this test commented out?

Reply:
This is a compatibility test, one should uncomment this line in order to serialize the model once again